tests/test_routes_agent_deploy.py

```
@@ -401,3 +402,86 @@ async def test_archive_smoke_ok_false_on_empty_query(self, client):
+        assert host == "100.78.225.80"
+
+    async def test_online_worker_no_callback_host_500(self):
+        """A remote deploy with no reachable controller address must hard-fail,
+        not silently start an unreachable agent on 127.0.0.1."""
+        from types import SimpleNamespace
+        from tinyagentos.routes import agent_deploy
+        worker = SimpleNamespace(status="online", hardware={"arch": "x86_64"})
+        with patch.object(
+            agent_deploy, "controller_callback_host", new=AsyncMock(return_value=None),
+        ):
+            remote, host, err = await agent_deploy.configure_remote_deploy(
```

Ruff (0.15.18)

[warning] Unpacked variable `host` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)